ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/validator/batch/BatchedSubmissionValidator.scala

```diff
@@ -41,13 +39,15 @@ object BatchedSubmissionValidator {
       conflictDetection: ConflictDetection,
       metrics: Metrics,
       ledgerDataExporter: LedgerDataExporter,
+      logEntryIdComputationStrategy: LogEntryIdComputationStrategy,
```

Comment:
As normally you wouldn't change the log entry ID generation logic I suggest adding a default value.

Reply:
Also disagree. I have noticed that default values tend to get us into trouble because we don't question them. I'd rather have them explicit.

As the point of introducing this is to make it flexible, let's make it flexible.

Reply:
We need to think about consumers of this component as well, i.e., we need to make it easy for ledger integrators to do the right thing. If you insist on not introducing default parameters then let's create another factory method that uses sensible defaults (or at least provide guidelines in the scaladoc).

Reply:
I provided the default values.